test/example/typeinfo.txt

```diff
@@ -9,8 +9,8 @@ eq
 eqref
 (ref eq)
 i31
-(ref null i31)
 i31ref
+(ref i31)
```

Comment:
This change seems odd... did `i31ref` turn from nullable to non-nullable?

Reply:
It switched from non-nullable to nullable, but that was back in #4843. It looks like I made a drive-by fix to the printing in this PR when I added printing support for the new bottom types: https://github.com/WebAssembly/binaryen/pull/5115/files#diff-bff8588536e048ccdcbed7cb98bd8d3b564439785901aaa60bdfcdd8d0ac355eR2120-R2121.

Reply:
I see, thanks. Might be better in an earlier PR, though there isn't really much risk, so I don't feel strongly.